### DISCUSSION TITLE
Analyzing the GitHub marketplace

### DISCUSSION THREAD

**Discussion by giscus** [rajbos/rajbos.github.io#16] [Comments]
# Analyzing the GitHub marketplace

Analyzing the GitHub marketplace

https://devopsjournal.io/blog/2022/09/18/Analysing-the-GitHub-marketplace

**Comment by spindev** [comment]
Hi Rob, nice article and thanks for your work. 

The markdown of the third table looks broken :)

**Reply by rajbos** [reply]
Thanks. I've fixed it!
